TODO.md

```diff
@@ -0,0 +1,13 @@
+# TODO
```

Comment:
ich bin mir nicht sicher ob eine ToDo ins github muss

Reply:
Natürlich nicht. Da ich davon ausgehe, das einiges ohnehin aus diesem PR raus muss (zB Debug mode), habe ich mir in diesem ersten Commit nicht die Mühe gemacht. Ich entferne dies dann also noch.

Reply:
Done